week-04/README.md

```diff
@@ -34,6 +34,12 @@ The instructor will then review the pull request and make comments should furthe
 - *Question* (akrishnamurthy-stat6250): How should format name be defined for character and numeric data ?
 - Question (lceballos-stat6250): Why can't the format with the VALUE statement end in a period?
 - Question (aguenane-stat6250): What is the general form of the VALUE statement?
+- *Answer* (asharda-stat6250):
```

Comment:
Please update code sample with code fence; see https://help.github.com/articles/creating-and-highlighting-code-blocks/